robottelo/hosts.py

```diff
@@ -1092,6 +1093,10 @@ def install(self, installer_obj=None, cmd_args=None, cmd_kwargs=None):
             installer_obj = InstallerCommand(*cmd_args, **command_opts)
         return self.execute(installer_obj.get_command(), timeout=0)
 
+    def get_features(self):
+        """Get capsule features"""
+        return requests.get(f'https://{self.hostname}:9090/features', verify=False).text
```

Comment:
if this is only occurrence of using GET what about `curl -k <https..../features>`

but if we need more we need add the functionality to the Host object (if it is not already there)